src/WordPress_Object/Clarkson_Comment.php

```diff
@@ -101,8 +101,8 @@ public function get_meta( string $key, $single = false ) {
 	 *
 	 * @return bool|int
 	 */
-	public function update_meta( string $key, $value ) {
-		return update_comment_meta( $this->get_id(), $key, $value );
+	public function update_meta( string $key, $value, $prev_value ) {
```

Comment:
The `$prev_value` parameter should have the same fallback as the WP `update_comment_meta` function.
The default value is an empty string.

Reply:
In the function comment should be specified the parameter and the accepted types.